### DISCUSSION TITLE
ConEmu select box issue on high-DPI scaling

### DISCUSSION THREAD

**Discussion by Ricemug** [cmderdev/cmder#2916] [Q&A]
How to resolve the following problem where the height of the select box is reduced?
OS: Windows 11 Professional 23H2
Screen : 3840 * 2160 (Scale: 300% )
![image](https://github.com/cmderdev/cmder/assets/12928142/9a3aa8c2-5785-4ab9-a413-d26bef33b2c7)
![image](https://github.com/cmderdev/cmder/assets/12928142/6d1b31a3-8897-4a4a-9a5e-cc5000d38bca)

**Comment by daxgames** [comment]
This is not something we can help with since the dialog is from ConEmu, a component of Cmder.

**Comment by DRSDavidSoft** [comment]
@Maximus5 This is an issue with the DPI scaling of 300% as per indicated in the screenshot.
